examples/maps/maps.bal

```diff
@@ -28,19 +32,29 @@ public function main() {
     // Prints the number of mappings in the `map`.
     io:println(addrMap.length());
 
-    // Mappings can be removed using the `remove()` method.
-    boolean isRemoved = addrMap.remove("postalCode");
+    // Mappings can be removed using the `.remove()` method.
+    any isRemoved = addrMap.remove("postalCode");
```

Comment:
Shall we change the var name and type here? `remove()` returns the removed element and its type is the constraint type of the map